package.json

```diff
@@ -26,7 +26,6 @@
     "babel-preset-stage-3-without-async": "^1.0.0",
     "chai": "^4.1.2",
     "clipboard": "^2.0.1",
-    "domReady": "github:requirejs/domReady#d85bdc38a6df868ead52ab3e80715aaf60e68765",
```

Comment:
So, other profiles will still depend on it, but the W3C profile won't. So we should leave it as a dependency, otherwise we will break others.

Reply:
Hmm, do we have an example that uses `domReady`?

Reply:
ah, I was unaware that they had their own independent list of dependencies. I’d assumed they were using the same as us.

Reply:
Any forks should also have their package.json forked, so IMO they shouldn't be affected, no?

Reply:
The problem is merging https://github.com/Geonovum/respec/blob/develop/tools/copydeps.js

As it’s going to delete domRedady there, so things will break for them.

Reply:
About package.json, no unfortunately :( merging still occurs, so deletions still happen. Only new files on their forks are unaffected.

Reply:
Ah yes, deletions happen. I thought it will cause merge conflict for the profile but found that it's a separate one.

But even then they should have their CI on (or at least local test if not). Things can always be broken when merging, that's why we have tests...

Reply:
👍 for that reasoning.

Still we'll want to remove it after enough time, will be great if we can `mark as deprecated` or anything that will allow smooth migration. Otherwise we'll never be able to remove e.g. jQuery from package.json.

Reply:
We might have lead by example, and point other fork maintainers to our code so they can copy what we did. Eventually, we can force a breaking change... but yeah, some might be impossible to ditch, like jQuery... but hopefully we will eventually fully remove it from core/ and w3c/.